jenkins/pipelines/ci/tidb/tidb_ghpr_build.groovy

```diff
@@ -67,7 +67,7 @@ VOLUMES = [
 
 def user_bazel(branch) {
     // set the feature branch at here.
-    if (branch in ["master"]) {
+    if (branch in ["master", "feature/distribute-reorg"]) {
```

Comment:
调整下正则匹配 所有的 feature/* 或者 feature_*

Reply:
==~ /^feature[\/_].*/

Reply:
用 startsWith 也可以

Reply:
I think it is OK that it can use the other mirror If Someone would like to create a feature branch for update the old release. BTW, we can know who is using feature branch CI so that we can forecast the number of machines in the CI.